xml/monitoring.xml

```diff
@@ -111,6 +113,499 @@
    For more information about options, see the man page of ConMan.
   </para>
  </sect1>
+ <sect1 xml:id="sec-monitoring-clusters-prometheus">
+  <title>Monitoring &hpc; clusters</title>
+  <para>
+   You can monitor clusters in &productname; using &prometheus; and &grafana;.
+  </para>
+  <para>
+   &prometheus; collects metrics from exporters that run on the machines in the
+   cluster, and stores the data in a time series database. &grafana; provides
+   data visualization dashboards for the metrics collected by &prometheus;.
+   You can download pre-configured dashboards from the &grafana; website.
```

Comment:
Maybe we should have a link to said website here?

Reply:
I was going to say "That's already in More Information", but then remembered that's just a link to the docs and not the dashboards site. So yes, good suggestion!